app/tests/mocks/storage.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function () {
+  'use strict';
+
+  function StorageMock (options) {
+    options = options || {};
+
+    this.isSessionStorageEnabled = function () {
```

Comment:
Another possible approach is to create a `Storage` instance and override these functions as needed using sinon.